e2e_samples/unstructured_data/scripts/run_experiments.py

```diff
@@ -111,20 +96,18 @@
     telemetry = args.telemetry.upper() if args.telemetry is not None else None
     configure_telemetry(telemetry)
 
-    data_path = RepoPaths.data_path(args.data_filename)
-    experiment_config = RepoPaths.experiment_config_path(args.experiment_name)
+    data_path = Path(__file__).absolute().parent.parent.joinpath("data").joinpath(args.data_filename)
 
     run_results = run_experiments(
-        config_filepath=experiment_config,
+        config_filepath=args.experiment_config_path,
         variants=args.variants,
         run_id=args.run_id,
         data_path=data_path,
-        output_dir=RepoPaths.run_outputs,
+        write_to_file=True,
```

Comment:
The output dir is controled by the Config which can be updated through encironment variables. Now we just need to specify if we want to right to a file instead providing the output directory here. This allows the orchestrator to read and write from the same known directories without the user needing  to supply the same variable mutiple times